app/templates/suppliers/_frameworks_coming.html

```diff
@@ -0,0 +1,12 @@
+{% for framework in frameworks.coming %}
+  {%
+    with
+    messages = [
+      "We’ll email you when you can apply to Digital Outcomes and Specialists.",
```

Comment:
This should either be generic, e.g. `{{ framework.name }}` instead of  `Digital Outcomes and Specialists` - and similarly the URL should use `{{ framework.slug }}, or if it is DOS-only the conditional should be something more like `if DOS in frameworks.coming ...`